services/integration/src/routes/projects.js

```diff
@@ -0,0 +1,10 @@
+const Router = require('express').Router();
+const project = require('../controllers/project');
+
+Router.post('/', project.create);
+Router.get('/:id', project.getOne);
+Router.get('/', project.getAll);
+Router.put('/:id/update', project.update);
+Router.delete('/:id/delete', project.deleteOne);
```

Comment:
This is not REST compliant. The only thing you need to differentiate the operation you are performing is the HTTP verb, the `delete`, `update` part in the routes are not necessary.